kubernetes/docs/includes/deploy-jars-k8s.rst

```diff
@@ -0,0 +1,36 @@
+
+This demonstration utilizes a `custom Connector to generate mock events <https://github.com/confluentinc/kafka-connect-datagen>`__ which is **not** installed by default in the Confluent Kafka Connect Docker images.  In order to utilize this plugin, we build a custom Docker image which derives from the base Confluent Operator based Docker image and install our the custom connector using the ``confluent-hub`` client.
+
+Here is the ``Dockerfile`` snippet used to build these images (`source <https://github.com/confluentinc/kafka-connect-datagen/blob/0.1.x/Dockerfile-operator-local>`__):
+
+::
+
+  ARG CP_VERSION
+
+  FROM confluentinc/cp-server-connect-operator:${CP_VERSION}.0
+
+  ARG KAFKA_CONNECT_DATAGEN_VERSION
+
+  COPY target/components/packages/confluentinc-kafka-connect-datagen-${KAFKA_CONNECT_DATAGEN_VERSION}.zip /tmp/confluentinc-kafka-connect-datagen-${KAFKA_CONNECT_DATAGEN_VERSION}.zip
+
+  RUN confluent-hub install --no-prompt /tmp/confluentinc-kafka-connect-datagen-${KAFKA_CONNECT_DATAGEN_VERSION}.zip
+
+The `confluentinc/cp-server-connect-operator <https://hub.docker.com/r/confluentinc/cp-server-connect-operator>`__ image is used as the Base image upon which the ``confluent-hub`` client is used to install the custom connector.   See the documentation on `confluent-hub client <https://docs.confluent.io/current/connect/managing/confluent-hub/client.html>`__ for details for `installing from Confluent Hub <https://docs.confluent.io/current/connect/managing/confluent-hub/client.html#installing-components-with-c-hub-client>`__ or from a `local archive <https://docs.confluent.io/current/connect/managing/confluent-hub/command-reference/confluent-hub-install.html#confluent-hub-client-install>`__.
+
+Once the Docker image is built with your custom archive installed, Kubernetes will need to be able to pull this image from a Docker Registry to create the Pods.  The demonstration accomplishes this by publishing the image to `Docker Hub <https://hub.docker.com/r/cnfldemos/cp-server-connect-operator-with-datagen>`__.
+
+Finally your Operator Helm values will need to be updated to pull the custom images for your Pods.  This demo accomplishes this by overriding the ``connect`` image repository like so:
+
+::
+
+  connect:
+  image:
+    repository: cnfldemos/cp-server-connect-operator-with-datagen 
+    tag: 0.1.6-5.3.1.0
+
+See the demo's `value.yaml <https://github.com/confluentinc/examples/blob/5.3.1-post/kubernetes/gke-base/cfg/values.yaml#L53>`__ file and the `kafka-connect-datagen <https://github.com/confluentinc/kafka-connect-datagen>`__ GitHub repository for the full example.
```

Comment:
This will vary depending on the demo.  Consider either:

1. Adding a link to both
2. Removing the link from the include and add to each file below

Reply:
It's the same across demos, demo 2 inherts demo 1.  This is how it was when the language was in demo 2 originally before including it across demo 1

Reply:
My bad.